src/parser/jobs/pld/modules/Requiescat.tsx

```diff
@@ -15,27 +15,28 @@ const SEVERITIES = {
 		1: SEVERITY.MEDIUM,
 		5: SEVERITY.MAJOR,
 	},
-	MISSED_BUFF_REQUIESCAT: {
+	MISSED_CONFITEORS: {
 		1: SEVERITY.MAJOR,
 	},
 }
-const EXPECTED_REQUIESCAT_CASTS = 4
+const EXPECTED_REQUIESCAT_CASTS = 5
 // When calculating rushing, adjust the start of the window by 1.5 seconds to allow for using Requiescat in the first weave slot
 const WINDOW_START_FORGIVENESS_FOR_RUSHING = 1500
-const REQUIESCAT_DURATION = 12000
+const REQUIESCAT_DURATION = 30000
 
 class RequiescatUsageEvaluator implements WindowEvaluator {
```

Comment:
This evaluator should probably be removed completely.  At a minimum, the copy on the suggestion needs to be updated so that it's talking about not using Confiteor instead of missing the Requiescat buff due to not enough MP, but the stock ExpectedActionsEvaluator is probably a better fit for tracking Confiteor and reporting on missed usages.

It might also be worth looking at changing this analyser to not use core BuffWindow and just kind of do its own tracking, so that we can also track the post-Confiteor combo as well in the same module (can we just call that the afterparty module, since it comes after the Confetti is deployed? 🤔)

Reply:
So, atm the Blades combo is being tracked by the core Combo module. If the PLD uses Confiteor and doesn't make it all the way to Valor, it flags as a broken combo. Would that need to be moved into this module to count as supported, or would that be post-supported improvement?

Reply:
I guess my thinking was that the Confiteor combo was strong enough to justify having its own separate suggestion.  But if you're happy with just the regular combo tracking for it then I'm fine to go without separate tracking / suggestion.

Reply:
Yeah, I'm not opposed to making it its own suggestion, but since the information is already being presented if the player makes a mistake, I thought it better to work on other modules and cycle back around to this when the other bases are covered.